internal/clients/team3/judge.go

```diff
@@ -22,15 +21,15 @@ func (j *judge) PayPresident() (shared.Resources, bool) {
 }
 
 // InspectHistory returns an evaluation on whether islands have adhered to the rules for that turn as a boolean.
-func (j *judge) InspectHistory(iigoHistory []shared.Accountability, turnsAgo int) (map[shared.ClientID]roles.EvaluationReturn, bool) {
-	outMap := map[shared.ClientID]roles.EvaluationReturn{}
+func (j *judge) InspectHistory(iigoHistory []shared.Accountability, turnsAgo int) (map[shared.ClientID]shared.EvaluationReturn, bool) {
+	outMap := map[shared.ClientID]shared.EvaluationReturn{}
 
 	// If we do not have sufficient budget to conduct the inspection,
 	// then we will return an empty map with true evaluations.
 	// if j.c.getLocalResources() < config.IIGOConfig.InspectHistoryActionCost {
 	// 	// dummy evaluation map
 	// 	for _, entry := range iigoHistory {
-	// 		outMap[entry.ClientID] = roles.EvaluationReturn{
+	// 		outMap[entry.ClientID] = shared.EvaluationReturn{
```

Comment:
Wait why do we have this random commented block XD

Reply:
lol i have no idea, shall i uncomment?